src/lib/UIKitConfigProvider/index.tsx

```diff
@@ -108,7 +108,12 @@ const UIKitConfigProvider = ({ storage, children, localConfigs }: UIKitConfigPro
   }, []);
 
   const configs = getConfigsByPriority(localConfigs, remoteConfigs);
-  return <UIKitConfigContext.Provider value={{ initDashboardConfigs, configs }}>{children}</UIKitConfigContext.Provider>;
+  const contextValue = useMemo(() => ({
```

Comment:
useMemo 는 없어도 되지 않을까요?

Reply:
I followed the instruction [here](https://react.dev/reference/react/useContext#optimizing-re-renders-when-passing-objects-and-functions). So basically I wanted to prevent unnecessary re-rendering where calls `useUIKitConfig`.

Reply:
앗 네 그러면 useMemo 안으로 `const configs = getConfigsByPriority(localConfigs, remoteConfigs)` 도 들어가야 할것 같습니다.

localConfigs 를 플랫하게 디펜던시 어레이에 넣던가 해야할 것 같은데 좋은 아이디어가 생각나지 않네요 🤔

Reply:
악 그러게요 getConfigByPriority 도 내부에 recursive 하게 호출되는 부분이 있어서 여기도 매번 새로운 함수만들고 하니 useMemo 내부에 넣어서도 딱히 perf 측면에서 이득이 있을지 모르겠네요. dependency array 에 localConfigs 를 넣는것도 그렇구요 😅

당장 필요한 perf optimize 가 아니니 일단 이 부분 제거했습니다. 추후에 이슈가 있을때 다시 고민해봐도 될것 같아요. 
https://github.com/sendbird/sendbird-uikit-react/pull/572/commits/8354d24492d6f83fed626491ca7918de20a17f20

Reply:
넵! 좋습니다.